src/sql/base/browser/ui/buttonMenu/buttonMenu.css

```diff
@@ -3,13 +3,10 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-.monaco-dropdown.button-menu {
+.in-preview .monaco-dropdown {
```

Comment:
Hmmm, you know we should actually consider setting this at the root workbench level. That would make stuff like this a lot easier in the long run if this is something we're going to be going with for the time being...

Reply:
I like that idea; that way I wouldn't have to add the conditional to a given component in order to apply this class.

Where do we begin?

Reply:
I'd look at how the theme styles are done and do something similar :

https://github.com/microsoft/azuredatastudio/blob/master/src/vs/workbench/services/themes/browser/workbenchThemeService.ts#L424

(it shouldn't be a part of the theme service though - it'd just be doing the same general thing of applying the class to the root container)